pycbc/events/coherent.py

```diff
@@ -59,9 +62,12 @@ def get_coinc_indexes(idx_dict, time_delay_idx):
                 [coinc_list, idx_dict[ifo] - time_delay_idx[ifo]]
             )
     # Search through coinc_idx for repeated indexes. These must have been loud
-    # in at least 2 detectors.
+    # in at least min_nifos detectors if the analysis uses more than 2 detectors.
     counts = np.unique(coinc_list, return_counts=True)
-    coinc_idx = counts[0][counts[1] > 1]
+    min_nifos_low = min_nifos - 1
+    coinc_idx = (
+        counts[0][counts[1] > min_ifos_low] if len(idx_dict) > 2 else counts[0]
+    )
```

Comment:
I am confused by the "if the analysis uses more than 2 detectors" part here. Did you mean more than 1 detector (consistently with the edits above)? If 2 is really what you meant, maybe it should be explained in the help string of `--nifo-sngl-snr-threshold`.

Reply:
Which implies: I am happy to remove it if you think it's just a wasteful slow down of things.

Reply:
Seems ok to keep it, though I would rather do
```
if len(idx_dict) == 1:
    return counts[0]
```
right after the call to `np.unique()`.

The first sentence of the docstring of this function also needs updating.